### PR TITLE
Add install instructions to README.md

### PR DESCRIPTION
**Pull request by natemcmaster**
Installing CLI command line tools continues to be a frequently asked question (cref #147, #266). This adds more detailed instructions for getting started.

cc @blackdwarf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,39 @@ DotNetTools
 [![Travis build status](https://img.shields.io/travis/aspnet/DotNetTools.svg?label=travis-ci&branch=dev&style=flat-square)](https://travis-ci.org/aspnet/DotNetTools/branches)
 [![AppVeyor build status](https://img.shields.io/appveyor/ci/aspnetci/DotNetTools/dev.svg?label=appveyor&style=flat-square)](https://ci.appveyor.com/project/aspnetci/DotNetTools/branch/dev)
 
-The project contains command-line tools for the .NET Core CLI.
+This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
```
Comment (glennc) [inline README.md]: I don't know that this is true anymore. There are no samples or anything in the Home repo now are there?

File: README.md [hunk 2]
```diff
@@ -4,10 +4,39 @@ DotNetTools
...
-This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+The repository contains command-line tools for the .NET Core CLI.
+
+ - [dotnet-watch](src/Microsoft.DotNet.Watcher.Tools/) (Microsoft.DotNet.Watcher.Tools)
+ - [dotnet-user-secrets](src/Microsoft.Extensions.SecretManager.Tools/) (Microsoft.Extensions.SecretManager.Tools)
+ - [dotnet-sql-cache](src/Microsoft.Extensions.Caching.SqlConfig.Tools/) (Microsoft.Extensions.Caching.SqlConfig.Tools)
+
+## How to Install
+
+Install tools by editing your \*.csproj file and adding a DotNetCliToolReference with the package name and version.
```
Comment (glennc) [inline README.md]: Typically we would backtick DotNetCliToolReference

**Comment by glennc** [comment]
Couple of minor comments. Are we assuming that people already know what these tools are by the time they get to this page? From the docs perhaps? Questioning whether a sentence about what each tool does at the beginning would be worthwhile.

File: README.md (hunk 1 above)
Comment (natemcmaster) [inline README.md]: Yeah, this line is probably outdated now. All of our repos had this in the readme, and all I did here was just rearrange the line. We should update aspnet/home.

File: README.md [hunk 3]
```diff
@@ -4,10 +4,39 @@ DotNetTools
...
 
- - [dotnet-watch](src/Microsoft.DotNet.Watcher.Tools/)
- - [dotnet-user-secrets](src/Microsoft.Extensions.SecretManager.Tools/)
- - [dotnet-sql-cache](src/Microsoft.Extensions.Caching.SqlConfig.Tools/)
+## Projects
 
-This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+The repository contains command-line tools for the .NET Core CLI.
+
+ - [dotnet-watch](src/Microsoft.DotNet.Watcher.Tools/) (Microsoft.DotNet.Watcher.Tools)
```
Comment (natemcmaster) [inline README.md]: FYI there are README's already for each of these tools that explain what each does.

**Comment by natemcmaster** [comment]
Updated to add more detail.

**Pull request merged**